lib/i18n_flat_yml_backend.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'i18n'
+require 'json'
+
+# Custom i18n backend that parse our "flat_yml" files into the nested
+# hash structure that i18n works with
+class I18nFlatYmlBackend < I18n::Backend::Simple
+  # @param filename [String] filename, assumed to have the locale slug in the filename ex "en.txt"
+  # @return [Array(Hash, Boolean)] tuple of a hash and keys_symbolized
+  def load_yml(filename)
```

Comment:
It looks like this method isn't used anywhere? Are we dealing with this in `I18nTasksFlatYml::FileFormatsOverrides#load_file` ?

Reply:
This is what overrides the superclass method

Reply:
Ah! I missed that this extended `I18n::Backend::Simple`. I was doing a simple code search in this project.